### PR TITLE
[T4A2][T15-A1]Zhou Fanyi

### PR DESCRIPTION
**Pull request by fanyiii**

### CODE REVIEW

**Comment by fanyiii** [comment]
ready for review

File: src/seedu/addressbook/data/person/Person.java [hunk 1]
```diff
@@ -10,12 +10,15 @@
  */
 public class Person implements ReadOnlyPerson {
 
+	private static int nextSequenceNumber = 1;
```
Comment (okkhoy) [inline src/seedu/addressbook/data/person/Person.java:13]: Notice that your indentation mechanism uses tabs for indentation. 
The coding standards we follow require using 4 spaces for indentation.
Tip: you can set Eclipse to do this for you, check out how!

File: src/seedu/addressbook/parser/Parser.java [hunk 2]
```diff
@@ -54,7 +54,7 @@ public Parser() {}
      * @param userInput full user input string
      * @return the command based on the user input
      */
-    public Command parseCommand(String userInput) {
+    public static Command parseCommand(String userInput) {
```
Comment (okkhoy) [inline src/seedu/addressbook/parser/Parser.java:57]: based on the files changed, I am guessing, you have not edited that part of the code where you create a parser object.
after making this method static, you can remove the parser object from your code!

**Comment by okkhoy** [comment]
@fanyiii 
Some comments added. Please ack & close the PR after reading comments.

**Pull request closed without merging**